rust/noosphere-ns/src/records.rs

```diff
@@ -202,15 +202,15 @@ impl FromStr for NSRecord {
         // Wait for next release of `ucan` which includes traits and
         // removes `try_from_token_string`:
         // https://github.com/ucan-wg/rs-ucan/commit/75e9afdb9da60c3d5d8c65b6704e412f0ef8189b
-        Ok(NSRecord::new(Ucan::try_from_token_string(ucan_token)?))
```

Comment:
Can remove this comment now with the new UCAN

Reply:
Might be some other changes via the addition of traits in ucan package, I can take a look separately

Reply:
If it's alright, I'll let you take care of this one for me :pray: